frappe/templates/includes/comments/comments.py

```diff
@@ -22,10 +22,15 @@ def add_comment(comment, comment_email, comment_by, reference_doctype, reference
 		frappe.msgprint(_('Comments cannot have links or email addresses'))
 		return ''
 
+	if comment_email:
+		if frappe.db.exists("User", comment_email) and \
+			frappe.session.user != comment_email:
+			frappe.msgprint(_('User already exists in the system. Please login and comment.'))
```

Comment:
Must be `frappe.throw`

Reply:
weird, thought i had changed it in the previous commit...